mwdb/web/src/components/RecentView/RecentView.js

```diff
@@ -102,6 +104,16 @@ export default function RecentView(props) {
                 return;
             setQueryError(error);
         })
+        api.getObjectCount(
```

Comment:
I would use that instead of `getObjectList` (preflight query), because it also checks if query is correct. Callbacks should be almost the same as in getObjectList with added setObjectCount before the setCurrentQuery.